src/preferences.html

```diff
@@ -81,7 +81,8 @@
             <div class="flex-box">
                 <p>Themes</p>
                 <select id="theme" name="theme">
-                    <option value="light" selected>Light</option>
+                    <option value="system-default" selected>System Default (experimental)</option>
```

Comment:
I'm wondering if we really need to be specific that this is an experimental option. In the worst case, light theme is used as fallback and nothing really wrong happens. @araujoarthur0, what do you think?

Reply:
I agree. Especially now that you've tested it on Mac too hahaha

Reply:
Two tests = production level! hahahah

@06b, can you please change this? My suggestion is:

```suggestion
                    <option value="system-default" selected>System Default</option>
```

With this, I think my previous comment on the width of the preferences window is not necessary anymore.